### PR TITLE
refactor(src): remove internal ramda reference

### PR DESCRIPTION
**Pull request by willblake01**
replace internal ramda _isObject

Ref #1361

# Pull request template

Please use the following [PR](https://github.com/char0n/ramda-adjunct/pull/905/files) as an example
of pull request. This PR deals with adding a new feature called `allSettledP`.

If you have any additional questions please don't hesitate to contact any of the core committers.

### CODE REVIEW

**Comment by willblake01** [comment]
@char0n Can you please review, thanks!

File: src/isPlainObj.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
-import _isObject from 'ramda/src/internal/_isObject';
 import { pipe, both, equals, toString, pathSatisfies, curryN } from 'ramda';
 
 import isNull from './isNull';
 import isObjLike from './isObjLike';
 import isFunction from './isFunction';
 
+const isObject = val =>
+  Object.prototype.toString.call(val) === '[object Object]';
```
Comment (char0n) [inline src/isPlainObj.js]:
```suggestion
```

File: src/isPlainObj.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
-import _isObject from 'ramda/src/internal/_isObject';
 import { pipe, both, equals, toString, pathSatisfies, curryN } from 'ramda';
 
 import isNull from './isNull';
 import isObjLike from './isObjLike';
 import isFunction from './isFunction';
 
+const isObject = val =>
```
Comment (char0n) [inline src/isPlainObj.js]: IMHO using following composition should do the trick too. Just to be consistent with other changes in other PRs. This suggestion should be completely equivavelnt.
```suggestion
const isObject = pipe(type, identical('Object'));
```

Comment (willblake01) [reply src/isPlainObj.js]: Sure thing, I like that. I think I did this one first or I probably would have gone with that the first time.

**Pull request merged**